README.md

```diff
@@ -1,2 +1,60 @@
-# identity-oauth2-grant-token-exchange
-WSO2 IS Token Exchange Grant Implementation
+# Welcome to the WSO2 Identity Server Token Exchange Grant Type for OAuth2. 
+
+This repository contains implementation for the OAuth 2.0 Token Exchange Grant, described in [spec](https://datatracker.ietf.org/doc/html/rfc8693).
+You can exchange external Identity Provider's token for the token issued by Identity Server. The initial implementation supports exchanging JWT type tokens.
+
+## How to test Token Exchange Grant Type:
+
+* Download and build the project by executing `maven clean install`
+* Copy the `grant.token.exchange-1.0.0.jar` into <Carbon-Home>/repository/components/dropins directory
+* Add the following configurations to deployment.toml to enable this grant type:
+```toml
+[[oauth.custom_grant_type]]
```

Comment:
Since we add this to be compatible with IS 5.11, we need to go with the above config options. When upgrading latest dependencies, we can read template it.